src/facade/cmd_arg_parser.cc

```diff
@@ -34,18 +34,29 @@ CmdArgParser::CheckProxy::operator bool() const {
   return true;
 }
 
-template <typename T> T CmdArgParser::NextProxy::Int() {
+template <typename T> T CmdArgParser::NextProxy::Num() {
```

Comment:
Next time please do this chnages in a separate PR, as this change requires a change in many other places in code which are not related to this task.
For this specific case the replica takeover did not use the parser so you could continue with this in this PR and make a new PR to do the changes in CmdArgParser. But if you have a change that you must to be able to use in your PR and this change involves refactoring other places in code please create a seprated PR for it and than rebase it to your branch.

Reply:
ok thx